Tasks/Maven/maventask.ts

```diff
@@ -203,6 +203,55 @@ function createMavenSQRunner(sqHostUrl, sqHostUsername, sqHostPassword, sqDbUrl?
     return mvnsq;
 }
 
+function processMavenOutput(data) {
+    data = data.toString();
```

Comment:
In my testing I have always seen it to be defined but wouldn't hurt to guard in the code